dask_planner/src/sql/logical/table_scan.rs

```diff
@@ -68,3 +170,45 @@ impl TryFrom<LogicalPlan> for PyTableScan {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::sync::Arc;
+
+    use datafusion::arrow::datatypes::{DataType, Field, Schema};
+    use datafusion_common::{Result, TableReference};
+    use datafusion_expr::{col, in_list, lit, logical_plan::table_scan, Expr};
+
+    use super::PyTableScan;
+
+    #[test]
+    fn dnf_inlist() -> Result<()> {
+        let schema = Schema::new(vec![Field::new("id", DataType::Int32, false)]);
+
+        // Dummy logical plan
+        let plan = Arc::new(
+            table_scan(TableReference::none(), &schema, None)
+                .unwrap()
+                .filter(col("id").eq(Expr::Placeholder {
```

Comment:
Just wanted to ask why an `Expr::Placeholder` is used here instead of an `Expr::InList`?

Reply:
Short answer: Laziness lol

Long answer: `PyExpr` currently requires a `LogicalPlan` instance to perform schema based queries. However, that is removed in #1085 so once that merges this snippet of code will cause a compilation error and need removed anyway. So knowing that I got a little lazy and just copied a `LogicalPlan` node from another test in the codebase.

You are right though, I should, and will update it to use `Expr::InList`